src/main/webapp/app/exercises/text/assess/conflicts/text-feedback-conflicts.component.ts

```diff
@@ -103,10 +103,12 @@ export class TextFeedbackConflictsComponent extends TextAssessmentBaseComponent
         await super.ngOnInit();
         if (!this.leftSubmission) {
             const submissionId = Number(this.activatedRoute.snapshot.paramMap.get('submissionId'));
-            const participation = await lastValueFrom(this.assessmentsService.getFeedbackDataForExerciseSubmission(submissionId));
```

Comment:
Line 107 is a downgrade from this line; toPromise will be [deprecated](https://indepth.dev/posts/1287/rxjs-heads-up-topromise-is-being-deprecated)